Assets/HoloToolkit/Input/Scripts/CursorManager.cs

```diff
@@ -24,32 +24,38 @@ public partial class CursorManager : Singleton<CursorManager>
 
     void Awake()
```

Comment:
While we're editing this file, could we make the Awake and LateUpdate methods private as well please?